packages/flutter_tools/lib/src/flutter_manifest.dart

```diff
@@ -519,7 +519,8 @@ void _validateFlutter(YamlMap? yaml, List<String> errors) {
           _validateFonts(yamlValue, errors);
         }
       case 'licenses':
-        errors.addAll(_validateList<String>(yamlValue, '"$yamlKey"', 'files'));
+        final (_, List<String> filesErrors) = _validateList<String>(yamlValue, '"$yamlKey"', 'files');
+        errors.addAll(filesErrors);
```

Comment:
I am ashamed of how often I accidentally wrote 

```dart
final (_, List<String> errors) = ...;
errors.addAll(errors);
```
and got a concurrent list modification runtime error.

Reply:
naming things be hard